src/sync/order_rewards.py

```diff
@@ -87,8 +87,7 @@ def sync_batch_rewards(
         block_from=last_sync_block(
             aws,
             table=sync_table,
-            genesis_block=16691686,  # First Recorded Batch Reward block
-            # TODO - use correct genesis block here. (note this is not actually determined yet)
+            genesis_block=16862919,  # First Recorded Batch Reward block
```

Comment:
cf https://github.com/cowprotocol/dune-sync/issues/31

I picked the block here!